### ISSUE TITLE
How to visualize the TSP trajectory?

### ISSUE THREAD

**Issue by christopherred** [Pointerformer/Pointerformer#3]
Maybe this question is kind of basic, but for a green hand like me, yes, actually I met this problem. The details are as follows:

Now I found data `s.selected_node_list` and `pi` are very likely to be the trajectory that I want, but it's a pity that I found they were not started by `1` and still weren't ended by 1 either.

So, where should I go to get a inferred sequence which is a real solution to a corresponding TSP problem by applying your trained model?

Best wishes!

**Comment by neo-pan** [comment]
You are correct that `s.selected_node_list` and `pi` represent the TSP trajectory. However, we use an `n` node list (instead of `n+1`) to represent the trajectory and did not guarantee that it starts or ends with node `1`. Two node lists that are shifted in order actually correspond to the same TSP trajectory. For example, `[0,1,2,3]` and `[1,2,3,0]` represent the same trajectory.

If you want a TSP trajectory that starts from `1`, you can shift it yourself.
